db.js

```diff
@@ -0,0 +1,5 @@
+const mongoose = require('mongoose');
+
+mongoose.connect(process.env.MONGO_URL).catch(err => {
```

Comment:
In you rebase you can add this to .env.example and pm2 config file

Reply:
Done